src/colmap/scene/database.cc

```diff
@@ -1846,6 +1860,8 @@ void Database::CreatePosePriorTable() const {
       "    position                   BLOB,"
       "    coordinate_system          INTEGER               NOT NULL,"
```

Comment:
nit: Shall we move the coordinate_system field above the position for consistency?